pyproject.toml

```diff
@@ -50,6 +50,7 @@ disallow_untyped_defs = false
 # Fully typed, enable stricter checks
 module = [
     "trio._abc",
+    "trio._core._asyncgens.py",
```

Comment:
The `.py` probably shouldn't be here, same with `_highlevel_open_tcp_stream`.

Reply:
Fixed in 07b26fa